universal-application-tool-0.0.1/app/assets/javascripts/main.ts

```diff
@@ -54,6 +54,20 @@ function maybeShowWarning() {
   }
 }
 
+ /** If a toast message is present, make it visible for the specified duration. */
+function maybeShowToast() {
+  const toastMessage = document.querySelector('.cf-toast');

```

Comment:
Question (blocking): is there any risk of there being multiple toasts?

Reply:
Yes. I already have that in the TODO section, but perhaps I should just implement it now.